developer/prog-guide/creating-a-windows-powershell-drive-provider.md

```diff
@@ -14,74 +14,69 @@ ms.assetid: 2b446841-6616-4720-9ff8-50801d7576ed
 caps.latest.revision: 6
 ---
 # Creating a Windows PowerShell Drive Provider
-This topic describes how to create a Windows PowerShell drive provider that provides a way to access a data store through a Windows PowerShell drive. This type of provider is also referred to as Windows PowerShell drive providers. The Windows PowerShell drives used by the provider provide the means to connect to the data store.
```

Comment:
What happened to this content?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.